app/javascript/components/pages/ItemNew.scss

```diff
@@ -0,0 +1,19 @@
+.section{
+  float: right;
+}
+
+.row-margin{
+  margin: 10px 5px;
+}
+
+.h1-styles{
+  font-style: "Montserrat", sans-serif;
+  color: white;
+  font-size: 5em;
+  text-shadow: 4px 4px 4px #49525A;
+  font-weight: bold;
+  font-family: "Montserrat",sans-serif;
+  line-height: 3.125rem;
+  letter-spacing: -1.1px;
+  padding: 50px;
```

Comment:
just be aware that for fixed units of measure like pixels (px), it doesn't scale the same way that rem or em would. This would be a concern when you feel your app on different devices/screen sizes